tool/config/linkcheck-skip-list.txt

```diff
@@ -37,4 +37,5 @@ https://material.io/guidelines/motion/transforming-material.html\#transforming-m
 
 # FIXME(Temporary): API entry that no longer exists. It is referenced from
```

Comment:
Do we really need this?

Reply:
Right, we don't anymore now that we have an issue to reference.